user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1990,6 +1990,15 @@ define([
     return path === REPOSITORY_ROOT_PATH;
   }
 
+  /**
+   * Escapes special characters in jquery selector
+   * @param selector
+   * @returns selector with special characters escaped by "\\"
+   */
+  function escapeJqSelector(selector) {
```

Comment:
I don't think there's anything jQuery specific in these selectors, so I'd call the function `escapeCssSelector`.
See, e.g., https://drafts.csswg.org/selectors-4/ and https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector.